jar-persistence/src/main/java/com/sixsq/slipstream/persistence/Vm.java

```diff
@@ -60,12 +60,14 @@
 @NamedQueries({
 		@NamedQuery(name = "byUserAndCloud", query = "SELECT v FROM Vm v WHERE v.user_ = :user AND v.cloud = :cloud"),
 		@NamedQuery(name = "usageByUser", query = "SELECT v.cloud, COUNT(v.runUuid) FROM Vm v WHERE v.user_ = :user AND v.isUsable = 1 AND v.runUuid IS NOT NULL AND v.runUuid <> 'Unknown' GROUP BY v.cloud ORDER BY v.cloud"),
```

Comment:
I see `usageByUser` named-query is no longer used.  Shouldn't it be removed from the list?